pyaedt/modules/PostProcessor.py

```diff
@@ -3093,15 +3107,15 @@ def _create_fieldplot_line_traces(
         else:
             return False
 
-    @pyaedt_function_handler(objlist="objects", quantityName="quantity")
+    @pyaedt_function_handler(objlist="assignment", quantityName="quantity")
     def create_fieldplot_line(
-        self, objects, quantity, setup_name=None, intrinsics=None, plot_name=None, field_type="DC R/L Fields"
+        self, assignment, quantity, setup_name=None, intrinsics=None, plot_name=None, field_type="DC R/L Fields"
     ):
         """Create a field plot of the line.
 
         Parameters
         ----------
-        objects : list
+        assignment : list
             List of polyline to plot.
```

Comment:
```suggestion
            List of polylines to plot.
```
After the data type, should all these parameters have a comma followed by "optional" because their defaults are all ``None``?